skrub/_tabular_learner.py

```diff
@@ -106,24 +106,24 @@ def tabular_learner(estimator, *, n_jobs=None):
 
     We can easily get a default pipeline for regression or classification:
 
-    >>> tabular_learner('regressor')                                    # doctest: +SKIP
+    >>> tabular_learner('regression')                                    # doctest: +SKIP
```

Comment:
For the later changes here and under, I don't think that we need to change the documentation.